include/lib/lib_logger.hpp

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
+// Datadog, Inc.
+
+#pragma once
+
+#include <cstdarg>
+#include <cstdio>
+#include <mutex>
+
+namespace ddprof {
+template <typename... Args>
+void log_once(char const *const format, Args... args) {
```

Comment:
One of the const is pointless, though I was trying to get codeql to be OK with this.
I'm not sure why it complains considering this is const.